pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -517,7 +517,6 @@ func convertResolution(proxy *model.Proxy, service *model.Service) cluster.Clust
 			if service.Attributes.ServiceRegistry == provider.Kubernetes && features.EnableEDSForHeadless {
 				return cluster.Cluster_EDS
 			}
-
```

Comment:
I would encourage you to check out https://github.com/istio/istio/wiki/Development-Conventions#superfluous-changes. It would be better to focus on fixing real bugs users face (there are plenty in the issues tab). Thanks for your contributions!

Reply:
ok